Sources/ProjectDescription/FileListGlob.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+/// A type that represents a list of source files defined by a glob.
+public struct FileListGlob: Codable, Equatable {
```

Comment:
@fortmarek @pepicrft Is there any reason why we can't use the same type also in sources and resources (maybe part of anotherr PR)?

Reply:
There are additional params, so I agree to unify but, yes it's better in another PR

Reply:
yeah, the additional params would be a blocker at the moment. I like that this is not specific to headers, though, and we can potentially unify that 👍